pkg/agent/route/route_linux.go

```diff
@@ -1405,50 +1405,23 @@ func (c *Client) addVirtualNodePortDNATIPRoute(isIPv6 bool) error {
 	return nil
 }
 
-// AddLoadBalancer is used to add routing entry which is used to route LoadBalancer ingress IP to Antrea
-// gateway on host.
-func (c *Client) AddLoadBalancer(svcIP net.IP) error {
-	linkIndex := c.nodeConfig.GatewayConfig.LinkIndex
-	isIPv6 := utilnet.IsIPv6(svcIP)
-	var gw net.IP
-	var mask int
-	if !isIPv6 {
-		gw = config.VirtualServiceIPv4
-		mask = net.IPv4len * 8
-	} else {
-		gw = config.VirtualServiceIPv6
-		mask = net.IPv6len * 8
+// AddExternalIPRoute adds a route entry that forwards traffic destined for the external IP to the Antrea gateway interface.
+func (c *Client) AddExternalIPRoute(externalIP net.IP) error {
+	externalIPStr := externalIP.String()
+	if err := c.addServiceRoute(externalIP); err != nil {
```

Comment:
I think no need to extract this method given the current change, to reduce the diff.

Reply:
Done.